src/helm/mod.rs

```diff
@@ -114,7 +116,11 @@ mod test {
             .labels_mut()
             .insert("stackable.tech/demo".into(), "logging".into());
 
-        let output = std::fs::read_to_string("fixtures/helm/output.yaml").unwrap();
-        assert_eq!(serde_yaml::to_string(&values).unwrap(), output);
+        let expected = std::fs::read_to_string("fixtures/helm/output.yaml").unwrap();
```

Comment:
should the file path come in via `#[file(...)]` for `rstest`? Or is this a different usage than the ones that do?

Reply:
eg:

```rust
    #[rstest]
    fn test_serialize(
        #[files("fixtures/helm/input-required.yaml")] input: PathBuf,
        #[files("fixtures/helm/output.yaml")] output: PathBuf
    ) {
        let contents = std::fs::read_to_string(input).unwrap();
        let mut values: DynamicValues = serde_yaml::from_str(&contents).unwrap();

        values
            .labels_mut()
            .insert("stackable.tech/demo".into(), "logging".into());

        let expected = std::fs::read_to_string(output).unwrap();
...
}
```

Reply:
For now, there is only one output fixture file. I also don't know if `rstest` can currently "match" up multiple files so that the input file will be compared to the right output file.

For now, it should be fine as is. If we need more output files in the future, we can do some research on `rstest` and we might need to implement upstream changes.

Reply:
The example to mentioned will currently generate *two* test cases, one for each globbed file. In this case `test_input_required` and `test_output`. To enable the feature we want here, we definitely need upstream changes.